.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+name: build
+
+on: [pull_request, workflow_dispatch]
```

Comment:
maybe you want this running on push, i just copied something from another project and this what was set